api/src/main/java/ca/bc/gov/educ/graddatacollection/api/rules/demographic/DemographicStudentValidationIssueTypeCode.java

```diff
@@ -10,6 +10,11 @@
 
 public enum DemographicStudentValidationIssueTypeCode {
   STUDENT_PEN_BLANK("STUDENTPENBLANK", "PEN is blank. Correct PEN in system or through PEN Web.", ERROR),
+  STUDENT_ADDRESS_BLANK("STUDENTADDRESSBLANK", "Missing student address.", WARNING),
```

Comment:
Please drop the severity types from this class - they're not used.

Reply:
Removed them